test/web-platform-tests/to-run.yaml

```diff
@@ -0,0 +1,21 @@
+DIR: dom/collections
+
+HTMLCollection-as-proto-length-get-throws.html: [fail, https://github.com/jsdom/webidl2js/issues/62]
+HTMLCollection-supported-property-indices.html: [fail, Named versus indexed properties trickiness]
+HTMLCollection-supported-property-names.html: [fail, Names vs. ids priority and effect on enumeration order]
+domstringmap-supported-property-names.html: [fail, DOMStringMap not implemented]
+
+---
+
+DIR: dom/events
+
+AddEventListenerOptions-passive.html: [fail, https://github.com/tmpvar/jsdom/issues/1609]
```

Comment:
Could we place URLs to issues in a third (optional) index of the array? Keeping the short error descriptions and URLs separate could potentially be useful if we want to do more with the output in the future, and might help keep track of which failures are covered by existing bugs.

Reply:
Seems reasonable. Assuming yaml supports syntax like `[fail, , https://...]`? I.e. omitting array elements.

Reply:
It seems like it does, though I figure that all failures ought to have a short description (with or without a link) to make it easier to tell the cause just from reading the list.

Reply:
I guess so, just seems a bit annoying to have to duplicate the information. But I suppose it won't be that bad.